app/services/laa_reference_updater.rb

```diff
@@ -6,17 +6,19 @@ def initialize(contract)
   end
 
   def call
-    maat_reference = contract[:maat_reference]
+    @maat_reference = contract[:maat_reference] || dummy_maat_reference
```

Comment:
```suggestion
    maat_reference = contract[:maat_reference].presence || dummy_maat_reference
```
I think we should make this private as well

Reply:
👍